### PR TITLE
Brand ReadtheDocs

### PR DESCRIPTION
**Pull request by laceysanderson**
Issue #650 

## Type(s) of Change(s)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] API-specific change (fix or addition to an API function)
- [x] Updates documentation (inline or markdown files)

## Description

This PR brands our ReadtheDocs to match Tripal.info. Specifically, I changed the sidebar to match the tripal.info header, added our Logo and changed link and notice box colours.

## Testing?

** Assuming you have sphinx installed**
1. Pull this branch
2. Change to the docs directory
3. Run `make html`
4. Open `_build/index.html` to inspect the docs :-)

## Screenshots (if appropriate):
screen shot 2018-09-14 at 8 30 22 am
screen shot 2018-09-14 at 8 33 36 am
screen shot 2018-09-14 at 8 33 13 am
screen shot 2018-09-14 at 8 32 22 am
screen shot 2018-09-14 at 8 35 21 am
## Additional Notes (if any):

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/_static/theme_overrides.css [hunk 1]
```diff
@@ -4,12 +4,139 @@ See: https://rackerlabs.github.io/docs-rackspace/tools/rtd-tables.html
...
+  border-color: black;
+}
+.wy-nav-side {
+  color: #CCCCCC !important;
+  background: #2D2D34 !important;
+  background-image: url("hexagon_pattern.png") !important;
+  background-repeat: repeat !important;
+  background: -webkit-linear-gradient(left, $light-grey , $gray-base) !important;
+  background: -o-linear-gradient(right, $light-grey, $gray-base) !important;
+  background: -moz-linear-gradient(right, $light-grey, $gray-base) !important;
```
Comment (almasaeed2010) [inline docs/_static/theme_overrides.css]: Little tip: we dont' need `-o-linear-gradient` or `-moz-linear-gradient` any more https://caniuse.com/#feat=css-gradients

We do however, need `-webkit-linear-gradient`

File: docs/_static/theme_overrides.css [hunk 2]
```diff
@@ -4,12 +4,139 @@ See: https://rackerlabs.github.io/docs-rackspace/tools/rtd-tables.html
...
+}
+/* Main Content */
+.wy-nav-content-wrap {
+  background: whitesmoke;
+}
+a, a:hover {
+  color: #990000;
+}
+a:visited {
+  color: #cc0000;
```
Comment (almasaeed2010) [inline docs/_static/theme_overrides.css:82]: This is affecting the logo and turning it red when I think it'd be legible as white. Maybe we could add something like this:

```css
a.icon-home:visited {
  color: #ffffff;
}
```

**Comment by almasaeed2010** [comment]
This looks great :D. Thanks @laceysanderson! I only had a couple of comments on very minor changes but other than that I add my approval to this 👍

**Comment by laceysanderson** [comment]
Thanks for the feedback @almasaeed2010! I will implement this as soon as I get home to my laptop where I have sphinx working ;-P

**Comment by laceysanderson** [comment]
There, I applied both your fixes -Thanks again! I'll add the reviewer's approval tag for you based on your comment.

**Comment by spficklin** [comment]
This looks much better! Thanks Lacey.... Two approvals so I'm merging.

**Comment by laceysanderson** [comment]
Awesome :-D Thanks for the review @almasaeed2010 and @spficklin

**Pull request merged**